README.md

```diff
@@ -1,2 +1,171 @@
+
+<details>
+  <summary>Server</summary>
+  
+</details>
+
+<details>
+<summary>Database</summary>
+  
+</details>
+
+## Key Features <a name="key-features"></a>
+
+- **Header/Navbar(Provides an easy way to browse though the site)**
+- **Projects(Showcases different projects authored by me)**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://francis450.github.io/Njoroge)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+  -Basic understanding of HTML5 and CSS3.
+  -Git client installed
+  -Node Package Modules
+
+### Setup
+
+Clone this repository to your desired folder:
+
+`git clone https://github.com/francis450/Njoroge`
+
+
+### Install
+
+Install this project with:
+  `npm install`
+
+
+### Usage
+
+To run the project, execute the following command:
+
+- Click the  `Go Live` button at the bottom left part of Visual Studio Code.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author1**
+
+- GitHub: [@francis450](https://github.com/francis450)
+- Twitter: [@nkamandefrancis](https://twitter.com/nkamandefrancis)
+- LinkedIn: [Francis](https://linkedin.com/in/njoroge-francis)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+-  **Contact infomation section**
+-  **Footer**
+-  **Desktop Version**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please go and give this repository a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse and the entire team of code reviewers, thank you for your service. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+
+- **What errors should I look out for?**
+
+  - Html&CSS linters errors
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
- [ ] Feel free to take this piece of information out of your readme file as it does not pertain directly to your project